components/inputgroup/index.css

```diff
@@ -10,11 +10,453 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+  --spectrum-combobox-background-color-key-focus: var(--spectrum-gray-50);
+
+  --spectrum-combobox-background-color-disabled: var(--spectrum-disabled-background-color);
+  --spectrum-combobox-font-color-disabled: var(--spectrum-disabled-content-color);
+
+  --spectrum-combobox-border-color-invalid-default: var(--spectrum-negative-border-color-default);
+  --spectrum-combobox-border-color-invalid-hover: var(--spectrum-negative-border-color-hover);
+  --spectrum-combobox-border-color-invalid-active: var(--spectrum-negative-border-color-down);
+  --spectrum-combobox-border-color-invalid-key-focus: var(--spectrum-negative-border-color-key-focus);
+
+  --spectrum-combobox-alert-icon-color: var(--spectrum-negative-visual-color);
+}
+
+.spectrum-InputGroup--quiet {
+  --spectrum-combobox-min-inline-size: var(--spectrum-combo-box-quiet-minimum-width-multiplier);
+  --spectrum-combobox-spacing-block-start-to-border: var(--spectrum-field-edge-to-border-quiet);
+  --spectrum-combobox-spacing-inline-icon-to-button: var(--spectrum-combo-box-visual-to-field-button-quiet);
+  --spectrum-combobox-spacing-inline-start-to-textfield: var(--spectrum-field-edge-to-text-quiet);
+
+  --spectrum-combobox-border-color-disabled: var(--spectrum-disabled-border-color);
+
+  --spectrum-combobox-background-color: rgb(248, 248, 248);
+}
+
+@media (forced-colors: active) {
+  .spectrum-InputGroup {
+    --highcontrast-combobox-focus-indicator-color: Highlight;
+  }
+}
+
+.spectrum-InputGroup {
+  position: relative;
+
+  display: inline-flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+
+  min-inline-size: var(--mod-combobox-min-inline-size, var(--spectrum-combobox-min-inline-size));
+  block-size: var(--mod-combobox-block-size, var(--spectrum-combobox-block-size));
+
+  margin-block-start: var(--mod-combobox-spacing-label-to-combobox, var(--spectrum-combobox-spacing-label-to-combobox));
+
+  border-radius: var(--mod-combobox-border-radius, var(--spectrum-combobox-border-radius));
+  border-width: var(--mod-combobox-border-width, var(--spectrum-combobox-border-width));
+
+  .spectrum-InputGroup-textfield {
+    .spectrum-Textfield-validationIcon {
+      position: absolute;
+      inset-block-start: var(--mod-combobox-block-spacing-edge-to-alert, var(--spectrum-combobox-block-spacing-edge-to-alert));
+      inset-block-end: var(--mod-combobox-block-spacing-edge-to-alert, var(--spectrum-combobox-block-spacing-edge-to-alert));
+
+      /* TODO Apply icon size tokens correctly */
+      inline-size: var(--mod-combobox-icon-size, var(--spectrum-combobox-icon-size));
+      block-size: var(--mod-combobox-icon-size, var(--spectrum-combobox-icon-size));
+    }
+
+    &:focus, &:focus-visible, &.is-focused, &.is-keyboardFocused {
+      border: none;
+
+      &::after {
+        box-shadow: none;
+      }
+    }
+  }
+
+  .spectrum-InputGroup-button:not(:disabled):not(.is-invalid) {
+    .spectrum-PickerButton-fill {
+      /* Subcomponent token override */
```

Comment:
This means that the token below overrides subcomponent tokens, such as PickerButton in this case, that don't align with the Combobox designs. I also tried overriding these styles with selectors, but it turned out to be more complicated than overriding tokens.